content/special-topics/known-issues.md

```diff
@@ -0,0 +1,46 @@
+---
+title: known issues
+id: known-issues
+weight: 90
+draft: false
+author: "people"
+---
+
+Although darktable aims at keeping compatibility with old edits, there are some limitations that you need to be aware of and cannot be solved in a programmatic way.
+
+# History inconsistencies with modules enabled by default
+
+## Context
+
+The editing history of pictures processed in darktable is saved in the database and in the sidecar XMP files stored in regard of your RAW files, in the same folder. This history records every step of settings change that happened during editing and let you go back to any step at any time.
+
+Before darktable 3.0, modules like *white balance*, *demosaic* and *base curve* that were enabled by default for technical reasons did not write an entry in the editing history if the user did not change the default settings. The software always assumed that no history meant default settings were used. This is not safe because it assumes defaults will never change in the future and it prevents a systematic handling of modules.
```

Comment:
Minor note. You can use either but it's kind of become a convention that we use underscore for italic.